plugins/modules/postgresql_set.py

```diff
@@ -240,33 +240,57 @@ def pretty_to_bytes(pretty_val):
     # if the value contains 'B', 'kB', 'MB', 'GB', 'TB'.
     # Otherwise it returns the passed argument.
 
+    # It's sometimes possible to have an empty values
+    if not pretty_val:
+        return pretty_val
+
+    # If the first char is not a digit, it does not make sense
+    # to parse further, so just return a passed value
+    if not pretty_val[0].isdigit():
+        return pretty_val
+
+    # If the last char is not an alphabetical symbol, it means that
+    # it does not contain any suffixes, so no sense to parse further
+    if not pretty_val[-1].isalpha():
+        return pretty_val
+
+    # Extract digits
+    num_part = []
+    for c in pretty_val:
+        # When we reach the first non-digit element,
+        # e.g. in 1024kB, stop iterating
+        if not c.isdigit():
+            break
+
+        if c.isdigit():
+            num_part.append(c)
```

Comment:
conditions in lines 262 and 265 are inverses of each other. So why not just use an `else`?

Reply:
good spot! changed, thanks:)